src/components/Link/index.js

```diff
@@ -0,0 +1,33 @@
+import { Link as RouterLink } from 'react-router-dom'
+import classnames from 'classnames'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import './Link.scss'
+
+const Link = ({ children, className, href, routeTo, ...props }) => {
```

Comment:
I think `to` would be a better prop name, rather than `routeTo`, as it is still rendering a link component.

Reply:
Noted. I chose routeTo to make it obvious that you were using React Router. There's also the href component for outputting a plain `<a>` - perhaps it would be better to drop the `href` prop and determine the plain anchor from the presence of a protocol in the `to` prop instead? What do you think?

Reply:
Point being that I chose `routeTo` because there are two ways you can pass your destination in to the component, so I wanted to make it obvious which you were using. `href` is an established thing, so I figured it made sense to require the user to ask the component to "route to destination" if that's what they were looking to do. I'd rather just have one prop, of course!

Reply:
I think having 2 different props is fine but guess it would be easy to tell if it was an internal or external link from the presence of `http`? But renaming `to` to `routeTo` seems unnecessary.